src/index.ts

```diff
@@ -21,20 +24,46 @@ export const createKZG = async () => {
     const verifyBlobKzgProofWasm = module.cwrap('verify_blob_kzg_proof_wasm', 'string', ['array', 'array', 'array']) as (blob: Uint8Array, commitment: Uint8Array, proof: Uint8Array) => string
     const verifyKzgProofWasm = module.cwrap('verify_kzg_proof_wasm', 'string', ['array', 'array', 'array', 'array']) 
 
-    const loadTrustedSetup = (trustedSetup: TrustedSetup) => {
+    /**
+     * 
+     * @param trustedSetup - an optional trusted setup parameter provided by the user
+     * @returns 0 if loaded successfully or 1 otherwise
+     */
+    const loadTrustedSetup = (trustedSetup?: TrustedSetup) => {
+        if (trustedSetup === undefined) {
+            trustedSetup = mainnetTrustedSetup
+        }
         return loadTrustedSetupWasm(trustedSetup.g1, trustedSetup.n1, trustedSetup.g2, trustedSetup.n2)
     }
     
+    /**
+     * 
+     * @param blob - a blob of data formatted as a Uint8Array of 4096 bytes 
+     * @returns a KZG commitment corresponding to the input blob formatted as a 32 byte Uint8Array
```

Comment:
```suggestion
     * @returns a KZG commitment corresponding to the input blob formatted as a 48 byte Uint8Array
```
commitment is 48 bytes